microsoft-edge/devtools-guide-chromium/whats-new/2024/01/devtools-121.md

```diff
@@ -0,0 +1,125 @@
+
+![The Z-index tab in the 3D View tool](./devtools-121-images/z-index-tab.png)
+
+See also:
+* [Navigate webpage layers, z-index, and DOM using the 3D View tool](../../../3d-view/index.md)
+
+
+<!-- ====================================================================== -->
+## Correct Hide debugger or Show debugger icon is shown in Sources tool
+
+<!-- Subtitle: In previous versions of Microsoft Edge, the Hide debugger icon was displayed instead of the Show debugger icon and vice versa in the Sources tool. In Microsoft Edge 121, this issue has been resolved. -->
+
+The icon used to collapse or expand the **Debugger** pane on the right in the **Sources** tool was incorrect.  The **Hide debugger** icon was displayed instead of the **Show debugger** icon, and vice versa:
+
+![Icon in the Sources tool, before](./devtools-121-images/open-close-button-sources-before.png)
+
+In Microsoft Edge 121, the correct icon is now shown:
+
+![Icon in the Sources tool, after](./devtools-121-images/open-close-button-sources-after.png)
+
+See also:
+* [Sources tool overview](../../../sources/index.md)
+
+
+<!-- ====================================================================== -->
+## Added source map support in the Coverage tool
+
+<!-- Subtitle: Quickly identify and eliminate unused code with the improved source map support in the Coverage tool. -->
+
+Before Microsoft Edge 121, the **Coverage** tool only displayed information for generated files, and omitted source files even when a source map was attached.  The **Coverage** tool has now been improved so that when a generated file has a source map attached, the **Coverage** tool displays information for the source file as well as for the generated file.
+
+See also:
+* [Find unused JavaScript and CSS code with the Coverage tool](../../../coverage/)
+
+
+<!-- ====================================================================== -->
+## Advanced filtering capabilities in the Bottom-Up, Call Tree, and Event Log tabs in the Performance tool
+
+<!-- Subtitle: Use advanced filters (Match Case, Use Regular Expression, and Match whole word) in the Performance tool's Bottom-Up, Call Tree, and Event Log tabs, to give more focused results. -->
+
+In the **Performance** tool, the following advanced filtering buttons were added to the **Bottom-Up**, **Call Tree**, and **Event Log** tabs:
+* **Match Case**.
+* **Use Regular Expression**.
+* **Match whole word**.
+
+![The 'Match Case', 'Use Regular Expression', and 'Match whole word' buttons in the Performance tool's Bottom-Up, Call Tree, and Event Log tabs](./devtools-121-images/advanced-filters.png)
+
+Also, in the **Bottom-Up** tab, the filter has been updated to now only apply to the top-level items, instead of matching every node.
+
+See also:
+* [Introduction to the Performance tool](../../../evaluate-performance/index.md)
+
+
+<!-- ====================================================================== -->
+## Tooling for PWA Window Controls Overlay
```

Comment:
```suggestion
## Window Controls Overlay section for PWAs, in Application tool's Manifest section
```
@leahmsft - which heading is best?
Alts:
* Window Controls Overlay section for PWAs, in Application tool's Manifest section
* Application tool's Manifest section adds a Window Controls Overlay section for PWAs
* Tooling for PWA Window Controls Overlay

Reply:
I kind of prefer "Tooling for PWA Window Controls Overlay" because that's likely going to be more recognizable to developers who've heard or used the feature.

Reply:
keeping as-is